.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "pfff"]
 	path = semgrep-core/pfff
-	url = https://github.com/returntocorp/pfff.git
+	url = https://github.com/IagoAbal/pfff.git
```

Comment:
you need to undo that. I've merged the div_by_zero so you should be able to reference the original pfff repo.
Note that Ruin0x11 is an extenral contributor and he managed to modify semgrep and pfff without 
referencing his own fork of pfff. Once you make a PR to the pfff repo, you can pull it and reference it from your semgrep repo (git pull origin; git checkout <branchname> where branchname is exactly the same name than the branch pushed to the main pfff repo).
If's some git gymnastic. Let me know next time you need that and I can help.

Reply:
Aha, it seems possible, although it's a bit more complicated than that: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally

Reply:
Thanks for the pointer, I'll do like that next time!